examples/flex-flows/basic/flow.flex.yaml

```diff
@@ -1,5 +1,10 @@
 $schema: https://azuremlschemas.azureedge.net/promptflow/latest/Flow.schema.json
 entry: flow:flow_entry
 environment:
-  image: mcr.microsoft.com/azureml/promptflow/promptflow-python
+  image: promptflow.azurecr.io/promptflow-runtime:pr-1316971-v19
   python_requirements_txt: requirements.txt
+environment_variables:
```

Comment:
Let's keep this in run.yaml, so flow test can actualy consume from env variable.

Reply:
Offline synced, will keep this in flow.flex.yaml so user can use env var in F5 but use connection in flow test